sktime/transformers/series_as_features/signature_based/_augmentations.py

```diff
@@ -0,0 +1,148 @@
+import torch
+from sklearn.pipeline import Pipeline
+from sktime.transformers.series_as_features.base import \
+    BaseSeriesAsFeaturesTransformer
+
+
+def get_augmentation_pipeline(aug_list):
```

Comment:
I'd call it `make_augmentation_pipeline` in line with sklearn's `make_pipeline` make func